### DISCUSSION TITLE
actor load failed: unknown actor code *

### DISCUSSION THREAD

**Discussion by nezha90** [filecoin-project/lotus#12711] [General Q&A] [answered]
network 24, actor 15, after the update, the actor cids can be obtained normally using "lotus state actor-cids --network-version=24", but unknown actor code bafk2bzacecnl2hqe3nozwo7al7kdznqgdrv2hbbbmpcbcwzh3yl4trog433hc will be reported when using *.load (such as miner.load)
My lotus version is 1.30

**Comment by hawkinggg** [comment, marked as answer]
maybe this will fix if you call *.load in your own code:   

> I suspect that it may be the way you’re using this externally, there were some things shifted around to avoid some internal entanglement and it means that certain use of lotus from outside won’t get you the actors loaded automatically if you needed them
you could try adding this to your imports and see if it fixes it, this will explicitly load the actors package and those builtin wasm actors will be available by CID:

`_ "github.com/filecoin-project/lotus/build"`

quote from  [rvagg](https://filecoinproject.slack.com/archives/CP50PPW2X/p1731974926098809?thread_ts=1731969061.607609&cid=CP50PPW2X) on slack

**Reply by nezha90** [reply]
Thank you, this method solved the problem
